lib/ansible/module_utils/facts/compat.py

```diff
@@ -65,8 +65,8 @@ def ansible_facts(module, gather_subset=None):
     filter_spec = module.params.get('filter', '*')
 
     minimal_gather_subset = frozenset(['apparmor', 'caps', 'cmdline', 'date_time',
-                                       'distribution', 'dns', 'env', 'fips', 'local', 'lsb',
-                                       'pkg_mgr', 'platform', 'python', 'selinux',
+                                       'distribution', 'dns', 'env', 'fips', 'iscsi', 'local',
```

Comment:
`iscsi` does not belong in the minimal target, Hardware or Network are more appropriate